.gitignore

```diff
@@ -140,6 +140,7 @@ include/gambit.h
 include/gambit-not*.h
 include/makefile
 lib/_eval.o
+lib/_gambit.js
```

Comment:
Might as well add the other universal backend targets:

lib/_gambit.py
lib/_gambit.php
lib/_gambit.rb
lib/_gambit.java
lib/_gambit.go

Reply:
Added in a new commit.